### DISCUSSION TITLE
Welcome to cordova-plugin-flic2 Discussions!

### DISCUSSION THREAD

**Discussion by bobobobo** [notnotse/cordova-plugin-flic2#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Marcosbar04** [comment]
Hello, i have this issue when i add the pluggin.
![image](https://user-images.githubusercontent.com/131439953/233578457-82305e97-08fe-401e-8d88-90b5d07e62aa.png)
If you can help me it, I would appreciate it very much.
Thanks.
